greenery/models/staging/stg_order_items.sql

```diff
@@ -0,0 +1,20 @@
+with source as (
+
+    select * from {{ ref('order_items_snapshot') }}
+
+),
+
+renamed as (
+
+    select
+        order_id::varchar as order_uuid,
+        product_id::varchar as product_uuid,
+        quantity::int,
+        dbt_valid_to is null as is_current_version,
```

Comment:
why don't you add dbt_valid_from ?

How would you use this staging model without it ? If you want to keep only the latest version, you also need to add a `WHERE` statement

Reply:
I guess this comment is relevant for all the other snapshots.

Reply:
I was waiting to have more information on downstream models to add additional fields. Regarding keeping only the last model, I was considering the option that downstream we might have a model with the versioning and perhaps a view only with the latest (although I'm not completely sure if this is somewhat a good practice)